projects/ontimize-web-ngx/src/lib/types/query-parameter.type.ts

```diff
@@ -0,0 +1,12 @@
+import { SQLTypes } from "../util/sqltypes";
+import { BasicExpression } from "./basic-expression.type";
+import { SQLOrder } from "./sql-order.type";
+
+export type QueryParameter = {
```

Comment:
If we want keep concordance with backend. the `QueryParameter` only has the attributes: filter, columns and sqltypes.
The `AdvancedQueryParameter` adds the attributes: offset, pageSize and orderBy